shell/platform/fuchsia/flutter/accessibility_bridge.h

```diff
@@ -105,28 +111,22 @@ class AccessibilityBridge
           OnAccessibilityActionRequestedCallback callback) override;
 
  private:
+  // Fuchsia's default root semantic node id.
+  static constexpr int32_t kRootNodeId = 0;
+
   // Holds only the fields we need for hit testing.
   // In particular, it adds a screen_rect field to flutter::SemanticsNode.
   struct SemanticsNode {
-    int32_t id;
-    int32_t flags;
-    bool is_focusable;
-    SkRect rect;
+    flutter::SemanticsNode data;
+    // int32_t id;
+    // int32_t flags;
+    // bool is_focusable;
+    // SkRect rect;
```

Comment:
commented out code (and below here)

Reply:
Concerning size of strings:
Note that the nodes are lazy  computed, and this is achieved by setting the callback in the a11y bridge constructor. This means that this code never runs, unless explicitly called in the command line by someone who is debugging a11y. So no change in size for there. I don't think there is a need to guard this behind a flag.

Concerning increase in memory:
We did a similar test recently for chrome, where we store the entire semantic tree in the a11y bridge, instead of only used fields. Given that chrome trees are much larger than flutter's, and there was no problem there, I am confident that this is not an issue here.

In our experience, chrome trees are two to three times bigger than flutter trees and have more data as well.

Reply:
sorry, I think kI answered the comment in the wrong thread -- but you will get it.. Will remove  the commented parts.

Reply:
For memory size: if this works for Chrome it's probably fine.

Binary size: I'm talking about the strings getting embedded in the binary we produce from this (e.g. in flutter_runner.far). There is a lot of translation logic in the new methods that goes from an enum to a string. All of those strings will have to be available in the binary now, even though we'd never expect them to be used in release. If we `#ifdef` them away that won't be an issue.

Reply:
I have been thinking about this one. Our goal is to be able to say, if an user says file a bug in any product, we want to dump the semantic tree and attach to the report. This means that this needs to be in the release, right? Because otherwise this usecase would not be possible.

So we had in mind:
1. User reports a bug or asks for help
2. Developer debugging
3. crash

In all these scenarios, we would like the inspect data to be available to  be attached to reports.

Reply:
Ahh, ok. If this needs to be in release though, would it need certain PII removed?

Reply:
Guarded the change behind a flag.